.github/workflows/darwin.yaml

```diff
@@ -0,0 +1,42 @@
+name: macos
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  build:
+
+    runs-on: macos-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v1 # Version range or exact version of a Python version to use, using SemVer's version range syntax
+    - uses: actions/setup-python@v2
```

Comment:
Do we really need Python on all platforms?

Reply:
Not sure, I think we need it on Linux tho - I'll do some testing later

Reply:
Huh, it doesn't seem like it's needed on any platforms - all of my builds on [this commit](https://github.com/Jabster28/edex-ui/commit/47da54be1a6e808051f01663eed8e282463eacf7) worked without it.

Reply:
Well great news then! One less thing that can break ;)